src/ingest-pipeline/airflow/dags/resource_map.yml

```diff
@@ -0,0 +1,76 @@
+# Order is important here!  The first regex pair that matches the DAG and task
+# ids will provide the entries for that DAG and task, so safe defaults defined
+# by wildcards for last.
+resource_map:
+  - 'dag_re': 'codex_cytokit'
+    'lanes': 2
+    'tasks':
+    - 'task_re': '.*cwl_cytokit'
+      'queue': 'gpu000_q1'
+      'threads': 6
+    - 'task_re': '.*'
+      'queue': 'general'
+      'threads': 6
+  - 'dag_re': 'celldive_deepcell'
+    'lanes': 2
+    'tasks':
+    - 'task_re': '.*cwl_segmentation'
+      'queue': 'gpu000_q1'
+      'threads': 6
+    - 'task_re': '.*'
+      'queue': 'general'
+      'threads': 6
+  - 'dag_re': 'ometiff_offset'
+    'lanes': 2
+    'tasks':
+    - 'task_re': '.*_exec_cwl1'
+      'queue': 'gpu000_q1'
```

Comment:
As far as I remember, the OME-TIFF offsets calculation doesn't require a GPU node; the CWL workflow is executed without any `--gpus` argument.